db/migrate/20230510122716_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
+    # add_index :likes, :user_id
```

Comment:
- [ ] It would be nice if you remove the commented piece of code to make your codebase clean and professional 👍 as required in the Ruby best practices 😊